### ISSUE TITLE
WCAG 2.2 Violation - There are meaningful (non-decorative) graphical elements with insufficient contrast present in the interface.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#2]
### Issue
There are meaningful (non-decorative) graphical elements present on the page that do not meet the minimum required contrast ratio of 3.000:1 with their background

#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- [1.4.11 Non-text Contrast](https://www.w3.org/TR/WCAG22/#non-text-contrast)

### User Impact

Some users (such as users with low vision, or users who are outdoors and experiencing glare on their screens) may be unable to see these meaningful (non-decorative) graphical elements.

#### Further information:

1. ["Web Accessibility Perspectives: Colors with Good Contrast" by W3C Web Accessibility Initiative (WAI) (YouTube) (links to the video in a broader playlist context)](https://www.youtube.com/watch?v=Hui87z2Vx8o&list=PLhDEeYUfW02QndusXXtQtuMbMYhK7TMBT&index=2)
2. ["Web Accessibility Perspectives: Colors with Good Contrast - Audio Described Version" by W3C Web Accessibility Initiative (WAI) (YouTube) (links to the video in a broader playlist context)](https://www.youtube.com/watch?v=a9kNUv6N8Rk&list=PLhDEeYUfW02Qo4r2KlzagxZxhYcZADee-&index=9)
3. [The "Intent" section of "Understanding Success Criterion 1.4.11: Non-text Contrast | WAI | W3C"](https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html#intent)

### Instances

- #26 

### Steps To Reproduce

Using a colour contrast checker tool such as [Accessible Color Picker](https://chromewebstore.google.com/detail/accessible-color-picker/bgfhbflmeekopanooidljpnmnljdihld):

1. Use the colour contrast checker tool's eyedropper tool to grab a colour value from the graphical element as well as the final, rendered background colour value, and calculate the contrast ratio between the the two values.
2. If the graphical element (or background) is dynamic (i.e. multiple colours), repeat this process, ensuring that you've tested all colour variations present in the graphical element and/or background.
3. Note that the graphical element does not meet the minimum required contrast ratio with its background (3.000:1).

### How To Fix The Issue

1. Update the graphical element's colour, the background colour, or both to meet (or exceed) the minimum required contrast ratio (3.000:1).
2. Repeat the "Steps To Reproduce" outlined above, verifying that the graphical element now meets the minimum required contrast ratio.

#### WCAG 2.2 Conformant Code Example

Not available/not applicable.
